components/multi-platform-controller/production/host-config.yaml

```diff
@@ -30,26 +30,26 @@ data:
   dynamic.linux-amd64.security-group: "launch-wizard-1"
   dynamic.linux-amd64.max-instances: "10"
 
-  host.power-rhtap-prod-1.address: "150.240.147.196"
+  host.power-rhtap-prod-1.address: "52.117.38.109"
   host.power-rhtap-prod-1.platform: "linux/ppc64le"
   host.power-rhtap-prod-1.user: "root"
-  host.power-rhtap-prod-1.secret: "ibm-ssh-key"
+  host.power-rhtap-prod-1.secret: "ibm-production-ppc64le-ssh-key"
   host.power-rhtap-prod-1.concurrency: "4"
 
-  host.power-rhtap-prod-2.address: "150.240.147.194"
-  host.power-rhtap-prod-2.platform: "linux/ppc64le"
-  host.power-rhtap-prod-2.user: "root"
-  host.power-rhtap-prod-2.secret: "ibm-ssh-key"
-  host.power-rhtap-prod-2.concurrency: "4"
+  # host.power-rhtap-prod-2.address: "52.117.38.122"
+  # host.power-rhtap-prod-2.platform: "linux/ppc64le"
+  # host.power-rhtap-prod-2.user: "root"
+  # host.power-rhtap-prod-2.secret: "ibm-production-ppc64le-ssh-key"
+  # host.power-rhtap-prod-2.concurrency: "4"
```

Comment:
This will be added, I just created the PR before the host finished provisioning.

Reply:
Updated